test/document.js

```diff
@@ -447,19 +448,33 @@ describe('serialize document', function() {
   });
 
   it('with invalid geopoint', function() {
-    const expectedErr = /Argument ".*" is not a valid number\./;
-
     assert.throws(() => {
-      new Firestore.GeoPoint('57.2999988', 'INVALID');
-    }, expectedErr);
+      new Firestore.GeoPoint(57.2999988, 'INVALID');
+    }, /Argument "longitude" is not a valid number/);
 
     assert.throws(() => {
-      new Firestore.GeoPoint('INVALID', '-4.4499982');
-    }, expectedErr);
+      new Firestore.GeoPoint('INVALID', -4.4499982);
+    }, /Argument "latitude" is not a valid number/);
 
     assert.throws(() => {
       new Firestore.GeoPoint();
-    }, expectedErr);
+    }, /Argument "latitude" is not a valid number/);
```

Comment:
I'm slightly against specifying exact error text in tests, but feel free to keep as is.